README.md

```diff
@@ -22,7 +22,8 @@ For more information about external storage in kubernetes, see
 
 ## Installation
 
-Recommended option is the [Helm Chart][helm chart].
+Recommended option is via [Helm][helm chart]
+```
```

Comment:
I think that's a leftover?
```suggestion
```

Reply:
Whoops let's squash merge this PR so this doesn't go into commit history 😅